modules/adprimeBidAdapter.js

```diff
@@ -0,0 +1,102 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
+
+const BIDDER_CODE = 'adprime';
+const AD_URL = 'https://delta.adprime.com/?c=o&m=multi';
```

Comment:
Responses from your server should be gzipped.

Reply:
This one done